doc/docs/master/hub/manage-entities/manage-workspaces.md

```diff
@@ -0,0 +1,73 @@
+# Manage Workspaces
```

Comment:
This doc does not have any section describing resources and autoscaling in Hub Pro. While there are not a lot of options for users to adjust here, it should describe how those are handled and managed and the limitations.